packages/jwt-middleware/README.md

```diff
@@ -0,0 +1,31 @@
+# JSON Web Token Middleware
+
+This middleware allows to secure execution of a webtask using JWT tokens by validating issuer, audience, and scope claims.
+
+## Usage
+
+1. Set the `wt-node-dependencies` metadata property to the stringified JSON of an object with names of modules as the keys and values set to the latest version for the corresponding module.
+
+```
+{
+    "@webtask/middleware-compiler": "1.3.0", 
+    "express-jwt": "5.3.1",
+    "jws": "3.1.4",
+    "jwks-rsa": "1.2.1"
```

Comment:
Since the middleware is packaged as its own NPM module, why does the webtask need to explicitly declare that middleware's dependencies? Can these dependencies be part of @webtask/middleware-compiler's dependencies in package.json, and therefore available when the webtask only takes a dependency on @webtask/middleware-compiler?

Reply:
Right now it is throwing an error when running the middleware if I do not declare the dependencies in the webtask, but I believe this is due to the fact that I am using a hosted `.js` file for the middleware as opposed to an NPM package (in progress).  I can revisit this when the NPM package is deployed.

Reply:
I am using this approach in https://github.com/tjanczuk/wtc#twitter-scheduler - the twitter scheduler compiler has a few module dependencies, but it declares them as its own NPM module dependencies. So the webtask only needs to take a dependency on the webtask ccompiler module itself to pull in the dependency closure. @ggoodman am I right?

Reply:
@tomek, you are right.

@lzychowski, since these dependencies are already in your `package.json`, they do not need to be manually added by users of this middleware. See: https://github.com/auth0/webtask-userspace/blob/a98f70b7f6f39e7d0f307e051550e93007f1c9f8/packages/jwt-middleware/package.json#L14-L18

Reply:
The list also **needs** to include this new package that will result: ```"@webtask/jwt-middleware": "1.0.0"```

In other words, both the middleware compiler and the middleware itself need to be listed as `wt-node-dependencies`.